src/azure-cli/HISTORY.rst

```diff
@@ -387,6 +387,10 @@ Release History
 
 * Enhanced error detail for vault delete in force mode.
 
+**AKS**
```

Comment:
history.rst is automatically generated now, pls add fix description in PR title. thanks.